src/main.c

```diff
@@ -11,16 +11,18 @@
 #include "../include/main.h"
 #include "../include/menu.h"
 #include "../include/randomizer_8_16_64.h"
-
+#include "../include/bubblesort.h"
 
 void randomizer_debug(){
     
     int **input = (int **)malloc(3 * sizeof(int *));
-    
     //inputs are accessible through 
     //input[0]...8, input[1]...16, input[2]...64
     randomizer_8_16_64(input);
-    
+    //Testing sort algorithm for all 3 array of 1.1 
+    bubblesort(input, 8);
+    bubblesort(input, 16);
+    bubblesort(input, 64);
```

Comment:
Für Debugging Anwendungen habe ich die Bubblesort Funktion in die Main eingefügt. Ideal wird in Konzernen allerdings  Module implementiert bei denen diese Debuggin Funktionen zusammengefasst getestet werden. Eine ähnliche Realisierung habe ich daher bei einem nächsten Pull request. Dient allerdings nur als Idee. Wenns euch gefällt können wir das ja so lassen ,ansonsten ändere ichs. Der Vorteil ist nur dass weniger im main.c ist. Da wir alle ja dort änderungen vornehmen, entstehen dort daher die meisten conflicts, mit so einer realisierung kann das reduziert werden, weil statt ganze blöcke wie hier rand_debug() nur einzelne funktionen im Debug Section eingefügt werden.